addons/mail/static/src/messaging/entity/messaging/messaging.js

```diff
@@ -155,6 +155,14 @@ function MessagingFactory({ Entity }) {
             autocreate: true,
             isCausal: true,
         }),
+        /**
+         * Mailbox History.
+         */
+        history: one2one('Thread'),
```

Comment:
feels like it should be auto create (but it lacks expressiveness due to default values) and probably causal too

Reply:
Indeed, but `autocreate` does not yet support passing data.